WordPress/src/main/java/org/wordpress/android/ui/blaze/BlazeUiState.kt

```diff
@@ -32,3 +32,54 @@ data class PostUIModel(
     val featuredImageUrl: String?
 ) : Parcelable
 
+sealed class BlazeWebViewHeaderUiState(
+    @StringRes open val headerActionText: Int,
+    open val headerActionEnabled: Boolean = true
+) {
+    @StringRes val headerTitle = R.string.blaze_activity_title
+
+    data class EnabledCancelAction(
+        @StringRes override val headerActionText: Int = R.string.cancel,
+        override val headerActionEnabled: Boolean = true
+    ): BlazeWebViewHeaderUiState(headerActionText, headerActionEnabled)
+
+    data class DisabledCancelAction(
+        @StringRes override val headerActionText: Int = R.string.cancel,
+        override val headerActionEnabled: Boolean = false
+    ): BlazeWebViewHeaderUiState(headerActionText, headerActionEnabled)
+
+    data class DoneAction(
+        @StringRes override val headerActionText: Int = R.string.done,
```

Comment:
🔍 Nitpick: the `done` word is in lower case. I think we should show **Done** , right now its **done**. 

![screenshot_nightMode_done_button_](https://user-images.githubusercontent.com/17463767/222176179-4419d528-853c-43ad-8064-0911fb26dcfc.png)

Reply:
Thanks @AjeshRPai - this is already on my list for the UI improvements PR, but thanks for making the issue visible to others.